File: trello-ladder/basic-string/reversewords.rb
# Given a string of words, write a function that returns a new string that contains the words in reverse order.

def reverse_words(string)
  string_array = string.split(" ")
  index = string_array.length - 1
  solution = ""
  while index >= 0
    if index == 0
      solution << string_array[index]
    else
      solution << string_array[index] + " "
    end
  index -= 1
  end
  solution
end

p reverse_words("popcorn is so cool isn't it yeah i thought so")

File: trello-ladder/basic-string/reversewords.js
// Given a string of words, write a function that returns a new string that contains the words in reverse order.

// Input: “popcorn is so cool isn’t it yeah i thought so”
// Output: “so thought i yeah it isn’t cool so is popcorn”

function reversewords(string) {
  var array = string.split(" ");
  var output = "";
  for (index = array.length - 1; index >= 0; index--) {
    output += array[index] + " ";
  }
  return output;
}

console.log(reversewords("popcorn is so cool isn’t it yeah i thought so"));
